### PR TITLE
blog about new dubbo admin

### PR DESCRIPTION
**Pull request by nzomkxia**
## What is the purpose of the change

XXXXX

## Brief changelog

XXXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Make sure there is a [GITHUB_issue](https://github.com/apache/incubator-dubbo-website/issues) filed for the change (usually before you start working on it). Trivial changes like typos do not require a GITHUB issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
- [ ] Format the pull request title like `Fix UnknownException when host config not exist #XXX`. Each commit in the pull request should have a meaningful subject line and body.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Test your code locally by running `docsite start`, and make sure it works as expected.
- [ ] Make sure no files under build directory is added.
- [ ] If this contribution is large, please follow the [Software Donation Guide](https://github.com/apache/incubator-dubbo/wiki/Software-donation-guide).

### CODE REVIEW

File: blog/zh-cn/dubbo-admin.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+* 对于服务级别的配置，会按照Dubbo2.6(URL)和Dubbo2.7(配置文件)两种格式进行写入，保证Dubbo2.6的客户端能够正确读取，解析规则 
+* 对于应用级别的配置，包括标签路由，只会按照Dubbo2.7的格式进行写入，因为Dubbo2.6无次功能，不需要做向前兼容。
+* Dubbo Admin只会按照Dubbo2.7的格式进行配置读取，因此，所有在Dubbo Admin上做的配置都可以被读到，但是之前遗留的，Dubbo2.6格式的URL无法被读取。
+* 对于同一个应用或者服务，每种规则只能够配置一条，否则新的会覆盖旧的。
+
+### 配置管理
+配置管理也是配合Dubbo2.7新增的功能，在Dubbo2.7中，增加了全局和应用维度的配置，
+* 全局配置： 
+![config](../../img/blog/admin/config.jpg)  
+全局配置里可以指定注册中心，元数据中心的地址，服务端和客户端的超时时间等，这些配置在全局内生效。除了配置写入，也可以用来查看。如果使用zookeeper作为注册中心和元数据中心，还可以看到配置文件所在位置的目录结构。
```
Comment (kexianjun) [inline blog/zh-cn/dubbo-admin.md]: 除了配置写入 should be 除了配置写入

File: blog/zh-cn/dubbo-admin.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+全局配置里可以指定注册中心，元数据中心的地址，服务端和客户端的超时时间等，这些配置在全局内生效。除了配置写入，也可以用来查看。如果使用zookeeper作为注册中心和元数据中心，还可以看到配置文件所在位置的目录结构。  
+* 应用， 服务配置  
+![appConfig](../../img/blog/admin/appConfig.jpg)  
+应用级别的配置可以为应用或者应用内的服务指定配置，在服务维度上，需要区分提供者和消费者。`dubbo.reference.{serviceName}`表示作为该服务消费者的配置，`dubbo.provider.{servcieName}`表示作为该服务提供者的配置。优先级服务 > 应用 > 全局。其中注册中心和元数据中心的地址，只能在全局配置中指定，这也是Dubbo2.7中推荐的使用方式。  
+
+### 元数据和服务测试  
+元数据是Dubbo2.7中新引入的元素，主要的使用场景就在Dubbo Admin中，主要体现在两个地方：  
+* 服务详情展示：  
+![metadata](../../img/blog/admin/metadata.jpg)  
+跟之前版本相比，Dubbo2.7中增加了对服务方法完整签名的记录，因此服务详情中也增加了方法信息的详情，可以看到方法名，方法参数列表以及返回值信息。
```
Comment (kexianjun) [inline blog/zh-cn/dubbo-admin.md]: Dubbo2.7 should be Dubbo2.7

File: blog/zh-cn/dubbo-admin.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+* 服务测试：   
+![test](../../img/blog/admin/test.jpg)
+更重要的，元数据为服务测试提供了数据基础，可以在也页面上调用真实的服务提供者，方便测试，也不需要为了调用服务去搭建一套Dubbo环境以及编写消费端代码。
```
Comment (kexianjun) [inline blog/zh-cn/dubbo-admin.md]: 可以在也页面上 also the same

**Comment by nzomkxia** [comment]
@kexianjun 不太理解你提的comment里面的不可读字符�是什么意思，是说我提交里面有乱码么？

**Comment by kexianjun** [comment]
> @kexianjun 不太理解你提的comment里面的不可读字符�是什么意思，是说我提交里面有乱码么？

是的，就是那个乱码的小方块，拷贝出来就成了�

**Comment by nzomkxia** [comment]
@kexianjun 谢谢指出，两篇都已修改

**Comment by lovepoem** [comment]
lgtm

**Pull request merged**